back/src/graphql/type/user.js

```diff
@@ -1,11 +1,11 @@
 import { conn } from '../../db/connection.js';
 import {
+  GraphQLBoolean,
```

Comment:
I don't know why their order changed, may be the result of `yarn format`...

Reply:
It's OK!